### ISSUE TITLE
Gem:ConflictError

### ISSUE THREAD

**Issue by lucatorella** [HazAT/badge#23]
I updated fastlane to 1.84, and I'm getting the following error now.

```
[10:52:52]: [32m-------------------[0m
[10:52:52]: [32m--- Step: badge ---[0m
[10:52:52]: [32m-------------------[0m
/usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:2284:in `raise_if_conflicts': Unable to activate badge-0.4.0, because mini_magick-4.5.1 conflicts with mini_magick (~> 4.0.2) (Gem::ConflictError)
```

I just have a lane with a badge step inside.

**Comment by elkraneo** [comment]
same here:
/Library/Ruby/Site/2.0.0/rubygems/specification.rb:2112:in `raise_if_conflicts': Unable to activate badge-0.4.0, because mini_magick-4.5.1 conflicts with mini_magick (~> 4.0.2) (Gem::ConflictError)

**Comment by NSCabezon** [comment]
How exactly did you solved it? I removed 4.0.4 and it crashes and with 4.5.1 same :/

**Comment by HazAT** [comment]
Should be fixed in new badge version 0.4.1
